source/_components/doods.markdown

```diff
@@ -0,0 +1,153 @@
+---
+title: "DOODS"
+description: "Detect and recognize objects with DOODS."
+ha_category:
+  - Image Processing
+ha_iot_class: Local Polling
+ha_release: 0.99
+redirect_from:
+ - /components/image_processing.doods/
+---
+
+The `doods` image processing platform allows you to detect and recognize objects in a camera image using [DOODS](https://github.com/snowzach/doods/). The state of the entity is the number of objects detected, and recognized objects are listed in the `summary` attribute along with quantity. The `matches` attribute provides the confidence `score` for recognition and the bounding `box` of the object for each detection category.
+
+## Setup
+
+You need to have DOODS running somewhere. It's easiest to run as a docker container and deployment is described on docker hub 
```

Comment:
Would be great if you could make an add-on of this docker container so that it is easy to set up in HA.